src/components/admin/user-accounts-tab.yml

```diff
@@ -90,3 +90,48 @@ userAccountsTab:
     de: |
       Wähle den Speicherplan aus, den du zuweisen möchtest. Keine Auswahl bedeuted
       Löschung {selectedItemCount, plural, =1 {eines im Benutzerkonto vorhandenen Speicherplans} other {von in den Benutzerkonten vorhandenen Speicherplänen}}.
+  roleInfoMarkdown:
+    en: |
+      # User
+
+      Users can create and edit public documents, as long as they are not protected or archived as well as upload public files.
```

Comment:
The whole part after the comma sounds a bit strange to me. And they actually can also create and edit private documents too 🤔 
How about "Users can contribute to the platform's content, with the restriction of protected or archived documents."? Hard to come up with something, as you experienced yourself probably.

Reply:
Changed according to your suggestions :-)

Reply:
🥇 🎉